metadrive/component/block/base_block.py

```diff
@@ -42,6 +48,8 @@ def __init__(
 
         self.block_index = block_index
         self.ignore_intersection_checking = ignore_intersection_checking
+        self.side_lane_line_type = side_lane_line_type
+        self.center_line_type = center_line_type
```

Comment:
Please make `self.side_lane_line_type = side_lane_line_type` and `self.center_line_type = center_line_type` be the properties of `PGBlock` instead of `BaseBlock`, as blocks like `ScenarioBlock` doesn't need these two properties.